changelog.txt

```diff
@@ -9,7 +9,7 @@
  * Updated ProGuard configuration. See documentation https://realm.io/docs/java/latest/#proguard for more details.
  * Updated Kotlin example to use 1.0.0-beta.
  * Fixed warnings reported by `lint -Xlint:all` (#1644).
- * Fixed a bug where simultaneous opening and closing a Realm on diffesrent threads might result in a NullPointerException (#1646).
+ * Fixed a bug where simultaneous opening and closing a Realm on different threads might result in a NullPointerException (#1646).
```

Comment:
opening and closing a Realm -> opening and closing of a Realm from different ...